sw/host/opentitanlib/src/io/async_help.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+
+/// In order to provide asynchronous functionality (e.g. in the `Uart` trait), some transport
+/// backends may need help from the main `mio` event loop.  This trait allows the transport to
+/// register its internal event sources with a poll registery used by the main loop, and the main
+/// loop will then be reponsible for calling `async_help` every time one of the transports internal
+/// sources become ready.
+pub trait AsyncHelp {
```

Comment:
Can we rename this file and the traits as something related to "nonblocking" so as to not confuse things with rust-lang's async/await features?

Reply:
Done